README.md

```diff
@@ -151,7 +151,3 @@ The SAML specification is a collection of PDFs (sadly):
 - [SAMLConformance](http://docs.oasis-open.org/security/saml/v2.0/saml-conformance-2.0-os.pdf) includes a support matrix for various parts of the protocol.
 
 [SAMLtest](https://samltest.id/) is a testing ground for SAML service and identity providers.
-
-## Security Issues
-
-Please do not report security issues in the issue tracker. Rather, please contact me directly at ross@kndr.org ([PGP Key `78B6038B3B9DFB88`](https://keybase.io/crewjam)). If your issue is *not* a security issue, please use the issue tracker so other contributors can help.
```

Comment:
We should update this part with our info, but it can be handled in a follow up PR.

Reply:
@mgyongyosi We have it here, I think we can just link to it https://github.com/grafana/saml/security/policy